qa/acceptance/spec/shared_examples/cli/logstash-plugin/install.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
```

Comment:
@ph lets make sure we open an issue to improve this, or add a comment to #5228, whatever if better.